movie-showcase/src/routes/+layout.svelte

```diff
@@ -0,0 +1,23 @@
+<a href="/"><div class="homebutton" /></a>
+<slot />
+
+<style>
+    :global(body) {
+        background: linear-gradient(
+            90deg,
+            rgba(199, 219, 247, 1) 0%,
+            rgba(191, 199, 222, 1) 35%,
+            rgba(197, 149, 148, 1) 100%
+        );
+        color: rgb(0, 0, 0);
+    }
+    :global(.title) {
+        font-size: 20px;
+        font-weight: bold;
+    }
```

Comment:
The `.title` class which is styled here does not exist in the HTML of this component. Therefore, it should be moved to the corresponding component.